lldb/source/Symbol/SwiftASTContext.cpp

```diff
@@ -1633,7 +1633,7 @@ lldb::TypeSystemSP SwiftASTContext::CreateInstance(lldb::LanguageType language,
                main_compile_unit_sp->GetPrimaryFile().GetCString());
   }
 
-  llvm::Triple triple = arch.GetTriple();
+  llvm::Triple triple = GetSwiftFriendlyTriple(arch.GetTriple());
```

Comment:
Why do we need to change the module context behavior to fix the REPL?

This is probably a good change, but it's a little scary since it also affects non-windows and usually triple changes break *something* :-)

Reply:
(The other CreateInstance function that creates an expression context is the one I would have expected that we need to change for the REPL)

Reply:
Ah, this is actually a cherry-pick (with fixup) of the change on main.  Let me test without the hunk and report back.

Reply:
Verified that we can get away without that for the REPL.